test/helpers/translationsProperties.js

```diff
@@ -1,10 +1,14 @@
 import translations from '../../translations/ui-agreements/en';
+import stripesSmartComponentsTranslations from '@folio/stripes-smart-components/translations/stripes-smart-components/en';
```

Comment:
Hmm, I see we would have to bring in `stripes-smart-components` as a devDependency (thats the lint warning you are facing) here just for the sake of translations in tests. Lets avoid that. Please revert this PR to the previous commit and we will go ahead with that version